File: pset6/mario.py
import cs50

def main():
    n = 0
    while True:
        print("Height: ", end = "")
        n = cs50.get_int()
        if(n >= 0 and n <= 23):
            break
        
    for i in range(n, 0 , -1):
        for a in range (i - 1):
            print(" ", end = "")
            
        for b in range(i, n + 1):
            print("#", end = "")
        print("#")
        
if __name__ == "__main__":
    main()
            

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n = 0;
    do {
       printf("Height: ");
       n = get_int(); 
    } while (n < 0 || n > 23);
     
    for(int i = n; i > 0; i--)
    {
        for(int a = 0; a < i - 1; a++)
        {
            printf(" ");
        }
        
        for(int b = i; b < n + 1; b++)
        {
            printf("#");
        }
        printf("#\n");
    }
    
}